### PR TITLE
Docs: Clean up 'win_service' module docs

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This is part of a series of module doc cleanups.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
win_service

##### ANSIBLE VERSION
v2.8

### CODE REVIEW

File: lib/ansible/modules/windows/win_service.py [hunk 1]
```diff
@@ -183,30 +196,30 @@
...
     state: restarted
     username: NT AUTHORITY\LocalService
-    password: ""
+    password: ''
 
 - name: Set dependencies to ones only in the list
   win_service:
     name: service name
-    dependencies: ['service1', 'service2']
+    dependencies: [ service1, service2 ]
```
Comment (jborean93) [inline lib/ansible/modules/windows/win_service.py:211]: should this be a yaml list instead, same with the other changes below?

Comment (dagwieers) [reply lib/ansible/modules/windows/win_service.py:211]: This is also a valid YAML list, both are possible.
I am not in favor to mandate using the expanded (multi-line) format.
In fact I think there's merit here in switching between both syntaxes in subsequent examples.

**Review by jborean93 — APPROVED** [review]

File: lib/ansible/modules/windows/win_service.py (hunk 1 above)
Comment (jborean93) [reply lib/ansible/modules/windows/win_service.py:211]: Fair enough, I do prefer the yaml format but that's a personal opinion, having both is a + for me.

Comment (dagwieers) [reply lib/ansible/modules/windows/win_service.py:211]: I'll put this on the list of things to discuss in the Documentation WG.
To me, the documentation examples serve a few purposes:
- Demonstrate possible ways to use the module first and foremost
- Show good practices (like always using `name:`)
- Learn possible constructs/syntax in YAML (e.g. like showing both YAML and JSON style lists)
- Learn possible constructs/syntax in Ansible (where useful add loops constructs or demonstrate registering output)
- Do not give wrong impressions (by using a minimalistic YAML syntax, e.g. indentation, quoting)

cc @acozine

**Pull request merged**